website/docs/workloads/mlperf/mlperf-profiles.md

```diff
@@ -33,9 +33,9 @@ Runs the MLPerf benchmark workload to test GPU performance.
   |-----------------------|---------|---------------|
   | Username     | Optional. User which needs to be created in container to run MLPerf benchmarks. | testuser  |
   | DiskFilter     | Optional. Filter to decide the disk that will be used to download benchmarks. Since benchmarks data is around 800gb, default disk filter is greater than 1000gb. | SizeGreaterThan:1000gb  |
-  | CudaVersion     | Optional. Version of CUDA that needs to be installed. | 11.6  |
-  | DriverVersion     | Optional. Version of GPU driver that needs to be installed. | 510  |
-  | LocalRunFile     | Optional. Link to download specified CUDA and GPU driver versions. | https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run  |
+  | LinuxCudaVersion     | Optional. Version of CUDA that needs to be installed. | 11.6  |
```

Comment:
I know we didn't have one document for the Linux driver installation, but could you help start one in website/docs/dependencies for this dependency class? You can add the Windows one first

Reply:
Added the documentation for Windows as well as Linux.